src/core/qgsuserprofilemanager.h

```diff
@@ -45,8 +45,12 @@ class CORE_EXPORT QgsUserProfileManager : public QObject
 
     /**
      * User profile manager used to manage user profiles for the instance of QGIS.
+     *
+     * If \a watchProfileFolder is true, then the specified \a rootLocation will be watched and any newly added
+     * profiles will cause profilesChanged() to be emitted. If \a watchProfileFolder is false, then the this will
+     * not occur.
```

Comment:
Can you add a very brief explanation of why you would (not) want the folder to be watched?